snapcraft.yaml

```
@@ -11,7 +11,7 @@
 
 name: webots
 title: Webots
-base: core20
+base: core18
 version: 'R2021a'
```

Note that in the final version we should also change this line
```suggestion
version: 'R2021b'
```